apps/pollbook/frontend/src/election_manager_screen.test.tsx

```diff
@@ -62,3 +65,55 @@ describe('Voters tab', () => {
     await screen.findByRole('heading', { name: 'Voter Details' });
   });
 });
+
+describe('SettingsScreen precinct selection', () => {
+  test('shows precinct select and allows changing configured precinct', async () => {
+    // Setup election with multiple precincts
+    const electionDef =
+      electionFamousNames2021Fixtures.readElectionDefinition();
+    const { precincts } = electionDef.election;
+    apiMock.setElection(electionDef);
+    apiMock.expectGetDeviceStatuses();
+    // Render
+    const renderResult = renderInAppContext(<ElectionManagerScreen />, {
+      apiMock,
+    });
+    unmount = renderResult.unmount;
+
+    // Wait for the SearchSelect to appear with the correct value
+    const select = await screen.findByLabelText('Select Precinct');
+    expect(select).toBeInTheDocument();
+    // Should have the correct initial value
+    expect((select as HTMLSelectElement).value).toEqual('');
+
+    // Simulate changing the precinct
+    const newPrecinctId = precincts[1].id;
+
+    apiMock.setConfiguredPrecinct(newPrecinctId);
+
+    userEvent.click(screen.getByText('Select Precinct…'));
+    userEvent.click(screen.getByText(precincts[1].name));
+
+    // Wait for the value to update
+    await vi.waitFor(() => {
+      screen.getByText(precincts[1].name);
+    });
+  });
+
+  test('does not show precinct select for single precinct election', () => {
+    // Setup election with multiple precincts
+    const singlePrecinctElection =
+      electionGridLayoutNewHampshireHudsonFixtures.readElectionDefinition();
+    apiMock.setElection(singlePrecinctElection);
+    apiMock.expectGetDeviceStatuses();
+    // Render
+    const renderResult = renderInAppContext(<ElectionManagerScreen />, {
+      apiMock,
+    });
+    unmount = renderResult.unmount;
+
+    // There should be no select precinct option shown
```

Comment:
nit: is it possible for this test to give a false pass if `Select Precinct` would exist, but the test exits before the render completes? should we check that render of the Election Manager screen completed before checking for absence of the precinct selector?